fineract-provider/build.gradle

```diff
@@ -157,18 +157,8 @@ openjpa {
     }
 }
 
-jacoco {
-    toolVersion = jacocoVersion
-    reportsDir = file("$buildDir/reports/jacoco")
-}
 
-jacocoTestReport{
-    reports{
-        html.enabled=true
-        xml.enabled=true
-        html.destination file("${buildDir}/code-coverage")
-    }
-}
```

Comment:
Removing this has no relation with your current issue. Any particular reason why you removed this?
This is not good